### ISSUE TITLE
json value get consumed after the string conversion

### ISSUE THREAD

**Issue by chanukaranaba** [ballerina-platform/ballerina-lang#19216]
**Description:**
Refer the following code;
```ballerina
            json jsonConversionRet = jsonutils:fromTable(selectRet);
            io:print("JSON: ");
            io:println(jsonConversionRet.toJsonString());
            io:println("--------------- ");
            io:print("JSON 2: ");
            io:println(jsonConversionRet);
```
JSON 2 value is blank, it means jsonConversionRet.toJsonString() consumed the stream and jsonConversionRet doesn't have value anymore. Is this is expected behavior?
From a developer perspective, the json value shouldn't be blank. 

**Steps to reproduce:**

**Affected Versions:**
1.0.0

**Comment by MaryamZi** [comment]
This is a bug with the `.toJsonString()` method. 

As @SupunS pointed out `getJSONString()` has not been implemented for `StreamingJsonValue`. This needs to be implemented adding https://github.com/ballerina-platform/ballerina-lang/blob/master/bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/StreamingJsonValue.java#L125.

**Comment by rdhananjaya** [comment]
Similar issue is present in `copy` function.
Please refer [ballerina-dev mail thread](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/ballerina-dev/zoPlcNMf_lQ/BAQciuTkCgAJ)

**Comment by rdhananjaya** [comment]
Similar issue is present in `copy` function.
Please refer [ballerina-dev mail thread](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/ballerina-dev/zoPlcNMf_lQ/BAQciuTkCgAJ)

**Comment by hasithaa** [comment]
Fixed with #19285
